recipe_batches/recipe_batches.py

```diff
@@ -1,14 +1,44 @@
 #!/usr/bin/python
 
+#Didn't need to use the math library
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  if len(recipe) is not len(recipe):
+    return 0
+  # Keeping track of how many times recipe divided by ingredients
+  count = []
+  # Iterate through recipe as the first loop because it set the standard
+  # ingredients needed to fulfill the recipe
+  for recipe_name, recipe_value in recipe.items():
+    #Iterate the ingredients and check for matching key values
+    for ingredient_name, ingredient_value in ingredients.items():
```

Comment:
nice! however, see the 2 nested for loops? this makes the solution O(n**2). see if you can find a way to bring it down to O(n)